jib-maven-plugin/src/integration-test/java/com/google/cloud/tools/jib/maven/BuildImageMojoIntegrationTest.java

```diff
@@ -266,22 +268,20 @@ private static String pullAndRunBuiltImage(String imageReference)
 
   private static void assertDockerInspectParameters(String imageReference)
       throws IOException, InterruptedException {
-    String dockerInspect = new Command("docker", "inspect", imageReference).run();
-    assertThat(dockerInspect)
-        .contains(
-            "            \"ExposedPorts\": {\n"
-                + "                \"1000/tcp\": {},\n"
-                + "                \"2000/udp\": {},\n"
-                + "                \"2001/udp\": {},\n"
-                + "                \"2002/udp\": {},\n"
-                + "                \"2003/udp\": {}");
-    assertThat(dockerInspect)
-        .contains(
-            "            \"Labels\": {\n"
-                + "                \"key1\": \"value1\",\n"
-                + "                \"key2\": \"value2\"\n"
-                + "            }");
+    String dockerInspectExposedPorts =
+        new Command("docker", "inspect", "-f", "'{{json .Config.ExposedPorts}}'", imageReference)
+            .run();
+    String dockerInspectLabels =
+        new Command("docker", "inspect", "-f", "'{{json .Config.Labels}}'", imageReference).run();
     String history = new Command("docker", "history", imageReference).run();
+
+    MatcherAssert.assertThat(
```

Comment:
minor nit: how about we make `MatcherAssert` static?

Reply:
Do you mean something like `import static org.hamcrest.MatcherAssert.assertThat`? (I wonder if this will introduce ambiguity with the static import for `com.google.common.truth.Truth.assertThat`, which is also present in this file.)

We might be able to remove this altogether if replaced by JSON comparison as @lqiu96 suggested below. I'll look into this as follow-up.